zio-sql/jvm/src/main/scala/Sql.scala

```diff
@@ -523,6 +537,21 @@ trait Sql {
     )(implicit ev: B <:< Integer): Expr[F :||: F2, A1, Integer] =
       Expr.Binary(self.widen[Integer], that, BinaryOp.OrBit)
 
+    def ~[A1 <: A]()(implicit ev: B <:< Integer): Expr.Unary[F, A1, Integer] =
```

Comment:
Unary operators can be written like this:

```scala
def unary_~ 
```

In order to be allowed to use them as unary operators in Scala.

Same for the other unary operators below.

Reply:
Thank you. I did not know. Changed using `unary_`